TUTORIAL.md

```diff
@@ -37,22 +43,30 @@ Running: End of Workflow
 Finished: SUCCESS
 ```
 
-## Flow scripts explained
 
-A workflow is a Groovy script which tells Jenkins what to do when your flow is run.
-If you are not familiar with Groovy, it is a scripting-friendly language related to Java ([documentation](http://groovy-lang.org/documentation.html)).
-You do not need to know much general Groovy to use Workflow; relevant bits of syntax will be introduced as needed.
 
-For this example, it suffices to know that `echo` is a _step_: a function defined in a Jenkins plugin and made available to all workflows.
-Groovy functions can use a C/Java-like syntax:
+
+
+
+
+## Understanding Flow Scripts
+
+A workflow is a [Groovy](http://groovy-lang.org/documentation.html) script that tells Jenkins what to do when your flow is run.
+You do not need to know much general Groovy to use Workflow - relevant bits of syntax are introduced as needed.
+
+**Example** In this example, `echo` is a _step_: a function defined in a Jenkins plugin and made available to all workflows.
+Groovy functions can use a C/Java-like syntax such as:
 
 ```groovy
 echo("hello from Workflow");
 ```
 
-but you can also drop the semicolon (`;`), drop the parentheses (`(` and `)`), and use single quotes (`'`) instead of double (`"`) if you do not need to perform variable substitutions.
+If you do not need to perform variable substitutions, you can:
+* drop the semicolon (`;`)
+* drop the parentheses (`(` and `)`)
+* use single quotes (`'`) instead of double (`"`)
```

Comment:
No, this is all wrong. `if you do not need to perform variable substitutions` applies only to the last item (kind of quotes). The first two are unrelated Groovy syntax simplifications. So you can have

```groovy
echo 'fixed text'
```

or

```groovy
echo "text with ${variables}"
```